util/HISTORY.md

```diff
@@ -4,10 +4,13 @@ History
 latest
 ------
 
+- Added f32 support to halo exchange data transformation
```

Comment:
Do we wait for a new version to categorize things as major or minor changes?

Reply:
Well I think this warrants a release, but I'd rather PR it afterwards